dashboard/db/migrate/20210617011456_set_published_state_attempt_two.rb

```diff
@@ -0,0 +1,43 @@
+class SetPublishedStateAttemptTwo < ActiveRecord::Migration[5.2]
+  def up
+    Script.all.each do |script|
+      script.update!({skip_name_format_validation: true, published_state: script.get_published_state})
```

Comment:
@davidsbailey Is there any concern that I'm planning to remove the get_published_state method in the next PR? Do I need to wait for people to migrate first before merging that PR?

Reply:
good question. yes, regardless of this shuffle, you'll need to leave `get_published_state` around for long enough for most engineers to run this migration before you remove it.